vendor.conf

```diff
@@ -0,0 +1,50 @@
+# grpc and protobuf
+google.golang.org/grpc v1.0.1-GA
+github.com/gogo/protobuf v0.3
+github.com/golang/protobuf 3c84672111d91bb5ac31719e112f9f7126a0e26e
+github.com/matttproud/golang_protobuf_extensions v1.0.0
+
+# etcd/raft
+github.com/coreos/etcd 3a49cbb769ebd8d1dd25abb1e83386e9883a5707
+github.com/coreos/go-systemd v12
```

Comment:
Nothing that fancy. I was just thinking something like this:

```
github.com/coreos/go-systemd v12 3a49cbb769ebd8d1dd25abb1e83386e9883a5707
```

That way, if upstream changes the tag, you know.